test/register-login.spec.js

```diff
@@ -0,0 +1,156 @@
+const { PageFactory } = require("../utils/page_factory");
+const { chromeConfig, sleep, quit } = require("../utils/webdriver/chrome_driver");
+const { expect } = require("chai");
+const { TestData } = require("../utils/test_data/test_data");
+
+describe("Test performs the testing of the login and sign up forms", () => {
+    before(async () => {
+        await chromeConfig();
+    });
+    after(async () => {
+        await sleep();
+        await quit();
+    });
+
+    it("Verify that page is loaded and the title matches the expected", async () => {
+        const homePageTitle = await PageFactory.getPage("Home");
+        await PageFactory.getPage("Home").open();
+        await PageFactory.getPage("Home").waitUntilTitleIsLoaded();
+        expect(await PageFactory.getPage("Home").getTitle()).equal(homePageTitle.expectedTitle);
+    });
+
+    it("Verify that cookie modal is opened and Accept button contains the text ", async () => {
+        const cookiePopup = await PageFactory.getPage("Cookie_Popup");
+        await PageFactory.getPage("Cookie_Popup").waitUntilElementIsLoaded(cookiePopup.cookiePopupModal);
+        expect(await PageFactory.getPage("Cookie_Popup").getElementText(cookiePopup.acceptButton)).equal(cookiePopup.acceptButton.locatorText);
+    });
+
+    it("Verify that modal is closed after submission", async () => {
+        const cookiePopup = await PageFactory.getPage("Cookie_Popup");
+        await PageFactory.getPage("Cookie_Popup").elementClick(cookiePopup.acceptButton);
+        expect(() => PageFactory.getPage("Cookie_Popup").getElement(cookiePopup.cookiePopupModal)()).to.throw();
```

Comment:
It's not a good practice to check Error being thrown while testing UI interfaces. It will be better to skip this test in your case, because if this popup is still on the screen all other tests will fail.